examples/v2.0/yaml/petstore-expanded.yaml

```diff
@@ -58,7 +58,7 @@ paths:
       description: Creates a new pet in the store.  Duplicates are allowed
       operationId: addPet
       parameters:
-        - name: pet
+        - name: body
```

Comment:
You can actually name the body parameter whatever you like. So 'pet' is perfectly legal in this case